report/devops-implementation.md

```diff
@@ -28,3 +28,9 @@
 
 #### Monitoring and Maintenance
 
+A key principle in DevOps is the ability to act based on metrics. To do this we use Micrometer metrics, a common monitoring facade which is vendor neutral, alike SLF4J is for logging. We use the Prometheus format as Quarkus provides a convenient library for combining these, `quarkus-micrometer-registry-prometheus`. To display and view these metrics we use Dashbuilder which allows easy visualization of the metric data via a YML format. Quarkus also has minimal config library for this, named `quarkus-dashbuilder`.
```

Comment:
"is for logging." Føler jeg ikke passer ind i sætningen

Reply:
explain pls

SLF4J ækvivalent til micrometer metrics, men SLF4J omhandler logging.